stdlib/public/core/SIMDVector.swift

```diff
@@ -927,7 +931,13 @@ extension SIMD where Scalar: FloatingPoint {
     // llvm.experimental.vector.reduce.fadd or an explicit tree-sum. Open-
```

Comment:
Nit to past-you: The first line of the implementation note has an elided word, "this eventually" probably needs a "will" or "should" somewhere in there.